honeycomb-core/src/utils/generation.rs

```diff
@@ -433,7 +409,9 @@ impl<T: CoordsFloat> GridBuilder<T> {
                 )
             }
             (_, _, _) => {
-                return Err(BuilderError::MissingParameters("Insufficient building parameters, please specify two out of three grid parameters"));
+                return Err(BuilderError::MissingParameters(
+                    "GridBuilder: insufficient building parameters",
```

Comment:
Maybe a dumb question, but why use `'a` lifetime and not `'static` on your `BuilderError` enum? Your error string looks to be compile-time known.   
Future-proofing?

Reply:
This is a correct observation, I don't really need the runtime lifetime for my errors atm.